### PR TITLE
UserService_DbContext_Fix

### PR DESCRIPTION
**Pull request by KostinP**
- Builder moved to entities
- Removed company entities
- New using declaration in UpdateDatabase

### CODE REVIEW

**Review by lalagvana — APPROVED** [review]
Well done :D

**Review by MihailKulikov — APPROVED** [review]

**Review by MihailKulikov — COMMENTED** [review]
Please, change using declaration in UpdateDatabase method in Startup.cs  for a new version.
This will reduce nesting.

**Review by Inzilbeth — APPROVED** [review]

**Comment by KostinP** [comment]
New structure DB:
UserService3

**Review by denis-kriv — APPROVED** [review]

File: DigitalOffice/Services/UserService/Startup.cs [hunk 1]
```diff
@@ -46,16 +46,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
         private static void UpdateDatabase(IApplicationBuilder app)
         {
```
Comment (mekk1t) [inline DigitalOffice/Services/UserService/Startup.cs]: Remove "static" modifier as we discussed it on the call. Other that that all's OK

**Review by mekk1t — APPROVED** [review]
Need to remove "static" from UpdateDatabase() and we're cool

**Pull request merged**